docs/examples/user-configuration/user-config-template.yaml

```diff
@@ -123,11 +152,20 @@ objects:
             limits:
               memory: 1Gi
           volumeMounts:
+            #
+            # Specify the directory that contains your custom configuration
+            # files. When you deploy the application this volume is mounted and
+            # the configuration files are copied to the container.
```

Comment:
I think we should say that the files are copied to the server's configuration directory as technically this volume is already mounted in the container.

Reply:
Also, instead of "Specify the directory" maybe say "Mount the directory that will contain your custom configuration" as this directory is not configurable, you have to use `/opt/datagrid/standalone/configuration/user`